app/appacme.py

```diff
@@ -32,7 +32,7 @@ def __init__(
         Create an account. As per acme standard an email needs
         to be provided.
         """
-        areq = {"termsOfServiceAgreed": True, "contact": ["mailto:email@example.com"]}
+        areq = {"termsOfServiceAgreed": True, "contact": ["mailto:helpdesk@irealisatie.nl"]}
```

Comment:
Would be nice if this is also configurable but for now this is fine as we are having more fixed values.

Reply:
I'll create a ticket for this

Reply:
https://github.com/minvws/nl-uzipoc-yubisign/issues/41